pkg/util/callback.go

```diff
@@ -34,16 +34,19 @@ func (c *callbackInvoker) ExecuteCallback(callbackURL string, success bool, labe
 	}
 	writer.Flush()
 
-	d := map[string]interface{}{
-		"labels":  labels,
+	data := map[string]interface{}{
 		"payload": buf.String(),
 		"success": success,
 	}
 
+	if len(labels) > 0 {
+		data["labels"] = labels
```

Comment:
With this check when build failed we don't send `labels` field at all (before it was `null`).